website/docs/guides/advanced-cluster-new-sharding-schema.html.markdown

```diff
@@ -0,0 +1,344 @@
+
+<a id="migration-replicaset"></a>
+## Migrating existing advanced_cluster type REPLICASET
+
+-> **NOTE:**  Please consider the following complementary documentation providing details on transitioning from a replicaset to a sharded cluster: https://www.mongodb.com/docs/atlas/scale-cluster/#convert-a-replica-set-to-a-sharded-cluster.
+
+Considering the following replica set configuration:
+```
+resource "mongodbatlas_advanced_cluster" "test" {
+    project_id   = var.project_id
+    name         = "ReplicaSetTransition"
+    cluster_type = "REPLICASET"
+
+    replication_specs {
+        region_configs {
+            electable_specs {
+                instance_size = "M10"
+                node_count    = 3
+            }
+            provider_name = "AZURE"
+            priority      = 7
+            region_name   = "US_EAST"
+        }
+    }
+}
+```
+
+To transition a replica set to sharded cluster 2 separate updates must be applied. First the cluster type must be adjusted to SHARDED, and apply this change without any other additional changes.
+
+```
+resource "mongodbatlas_advanced_cluster" "test" {
+    project_id   = var.project_id
+    name         = "ReplicaSetTransition"
+    cluster_type = "SHARDED"
+
+    replication_specs {
+        region_configs {
+            electable_specs {
+                instance_size = "M10"
+                node_count    = 3
+            }
+            provider_name = "AZURE"
+            priority      = 7
+            region_name   = "US_EAST"
+        }
+    }
+}
+```
+
+Once the cluster type is adjusted accordingly, we can proceed to add a new shard using the new schema:
+
+```
+resource "mongodbatlas_advanced_cluster" "test" {
+    project_id   = var.project_id
+    name         = "ReplicaSetTransition"
+    cluster_type = "SHARDED"
+
+    replication_specs { # first shard
+        region_configs {
+            electable_specs {
+                instance_size = "M10"
+                node_count    = 3
+            }
+            provider_name = "AZURE"
+            priority      = 7
+            region_name   = "US_EAST"
+        }
+    }
+
+    replication_specs { # second shard
+        region_configs {
+            electable_specs {
+                instance_size = "M10"
+                node_count    = 3
+            }
+            provider_name = "AZURE"
+            priority      = 7
+            region_name   = "US_EAST"
+        }
+    }
+}
+```
+
+<a id="leveraging-iss"></a>
+## Leveraging Independent Shard Scaling 
+
+Prerequisite: The new sharding schema must be used, meaning the advanced_cluster configuration is not using `num_shards` and therefore each shard is represented with a unique `replication_specs` element. Please refer to documentation above to transition into this schema if not already.
```

Comment:
Suggested rephrasing:

The new sharding schema must be used. Each shard must be represented with a unique `replication_specs` element. Please refer to the documentation above for details.

Reply:
sounds better, adjusted